pyproject.toml

```diff
@@ -6,7 +6,7 @@ build-backend = "setuptools.build_meta"
 description = "A Python client to control and data ports of the PandABlocks TCP server"
 name = "pandablocks"
 classifiers = [
-    "Development Status :: 3 - Alpha",
+    "Development Status :: 3 - Stable",
```

Comment:
This is not a valid classifier. See here: https://pypi.org/classifiers/

Reply:
Ah I suppose
`Development Status :: 5 - Production/Stable`
will do now